ts/dx.all.d.ts

```diff
@@ -674,7 +674,7 @@ declare module DevExpress {
      */
     resetOption(optionName: string): void;
 
-    _options: { silent(path, value): void };
+    _options: { silent(path: any, value: any): void };
```

Comment:
Not sure `dx.all` is allowed to be modified manually. @IlyaKhD, is it OK?

Reply:
Yep, there is a check for TS-bundle being up-to-date - no difference how it was updated, by tool, or by hand